packages/govern-server/src/core/config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "networks": {
```

Comment:
This should be an array cause it would simplify the code down in the network functions and the name wouldn't be duplicated.

Reply:
... of you keep the object and you remove the ``name`` property and it would just have to create once an ``array`` out of the ``object`` :-)